### PR TITLE
added k8s manifest

### PR DESCRIPTION
**Pull request by DipakKhade**
k8s manifest file for Next app, redis,postgres,Judge0 API and Judge0 workers

### CODE REVIEW

**Comment by SujithThirumalaisamy** [comment]
There is a main optimization you missed I guess. We need to autoscale the worker. Based on the redis queue length. CPU metric is fine. But the redis metrics will be the optimal thing

File: manifest.yaml [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+  name: judge0-worker-hpa
+  namespace: algo-100xdevs
+spec:
+  scaleTargetRef:
+    apiVersion: apps/v1
+    kind: Deployment
+    name: judge0-worker
+  minReplicas: 2
+  maxReplicas: 10
+  metrics:
```
Comment (SujithThirumalaisamy) [inline manifest.yaml:212]: This metrics should be the length of the queue

Comment (DipakKhade) [reply manifest.yaml:212]: ya i also realised that after

**Comment by siinghd** [comment]
#50 duplicate

**Pull request closed without merging**